pkg/asset/agent/image/agentimage.go

```diff
@@ -181,7 +182,10 @@ func (a *AgentImage) appendAgentFilesToInitrd(additionalFiles []string) error {
 	// Add a dracut hook to copy the files. The $NEWROOT environment variable is exported by
 	// dracut during the startup and it refers the mountpoint for the root filesystem.
 	dracutHookScript := `#!/bin/sh
-cp -R /agent-files/* $NEWROOT/usr/local/bin/`
+cp -R /agent-files/* $NEWROOT/usr/local/bin/
+# Fix the selinux label
+for i in $(find /agent-files/ -printf "%P\n"); do chcon system_u:object_r:bin_t:s0 $NEWROOT/usr/local/bin/$i; done`
```

Comment:
If `restorecon -F` worked and avoided having to explicitly specify the type, that seems preferable.

Reply:
Not sure if I've missed something, but in my tests `restorecon` wasn't available

Reply:
It's in CoreOS, but maybe not in the initramfs :slightly_frowning_face: